### ISSUE TITLE
"The Sky Is The Limit" (Astral Sorcery) Quest line needs updating/fixing

### ISSUE THREAD

**Issue by jestesared** [scalda/FoolCraft#676]
## SUGGESTION/HELP
- [ x] I would like to propose an additional mod, config change, custom recipe, etc.
- [ ] I am requesting help with installing or running the pack

## Issue Description
For the "Sky Is The Limit" quest line involving Astral Sorcery, some of the quests are unobtainable due to the nature of the mod itself. What I've notice so far is that quests for the Resonating Wand (A wand, a sorcerer's best friend #90), the Hand Telescope (Sky full of stars #114), and the Luminous Crafting Table (Next level crafting #91) are done through the crafting system in the mod, which is putting things in a crafting table modified by starlight or through an altar. Either way, they don't seem to work as 'crafting tasks' in terms of BetterQuesting, they should really be 'retrieval tasks.' Also, the quest for the telescope (#114, as mentioned above) is only obtainable after you get the Luminous Crafting Table (#91 as mentioned above), however, the quest for the Luminous Crafting Table is locked behind the telescope quest. Basically a catch 22. From what I can see right now, the other quests in this chain should be fine as they seem to retrieval tasks, but I haven't progressed further to know for sure.
For now, as I know that I've done the quests, I've just used the '/bq_admin complete <quest_id> [username:uuid]' command to complete the stuck quests.

## Reproduce Steps
Basically to reproduce you would just need to start through Astral Sorcery and try to go along with the quests for it at the same time.

**Comment by jestesared** [comment]
Those numbers are the quest numbers from the pack, I didn't realize they would hyperlink to other issues, my bad haha.

**Comment by DirtBottle** [comment]
Another issue with the quest line is that the rock crystal quest (questID = 115) does not ignore NBT which means it is not obtainable with most rock crystals mined.

**Comment by scalda** [comment]
thanks for this i am sorting this now

**Comment by scalda** [comment]
these have all now beed fixed thank you for the help
